src/System.Runtime/tests/System/GCTests.netcoreapp.cs

```diff
@@ -63,11 +63,10 @@ public static void GetGCMemoryInfo()
                     scenario = nameof(memoryInfo2.HighMemoryLoadThresholdBytes);
                     Assert.Equal(memoryInfo2.HighMemoryLoadThresholdBytes, memoryInfo1.HighMemoryLoadThresholdBytes);
 
-                    // TODO: the following test fails. Disabling now while figuring what  range of values is actually expected here.
-                    //       see: https://github.com/dotnet/corefx/issues/37378
-                    //
-                    //scenario = nameof(memoryInfo2.MemoryLoadBytes);
-                    //Assert.InRange(memoryInfo2.MemoryLoadBytes, memoryInfo1.MemoryLoadBytes, long.MaxValue);
+                    // Even though we have allocated, the overall load may decrease or increase depending what other processes are doing.
+                    // It cannot go above total available though.
+                    scenario = nameof(memoryInfo2.MemoryLoadBytes);
+                    Assert.InRange(memoryInfo2.MemoryLoadBytes, 1, memoryInfo1.TotalAvailableMemoryBytes);
```

Comment:
Is MemoryLoadBytes affected by the hard limit? TotalAvailableMemoryBytes is, correct?

Reply:
Yes. The implementation literally scales the recorded load to that total. Normally it is just physical memory. We do not expect/support the total to change.